libqtile/backend/wayland/xwindow.py

```diff
@@ -393,10 +406,20 @@ def _on_unmap(self, _listener: Listener, _data: Any) -> None:
         self.core.pending_windows.add(win)
 
     def _on_commit(self, _listener: Listener, _data: Any) -> None:
-        logger.debug("Signal: xstatic commit")
+        pass
+        # logger.debug("Signal: xstatic commit")
```

Comment:
Should we just drop this method and its listener?

Reply:
Fine with me.  Each instance of gkrellm was producing 10 lines per second in the log, which is why I had it commented.